tests/unit/general/bin-common.bats

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bats
+
+# bats file_tags=static,general,bin:common
+
+setup_file() {
+  load "../../bats.lib.bash"
+  load_common "env.bash"
+  load_common "gpg.bash"
+  load_common "yq.bash"
+
+  gpg.setup
+  env.setup
+
+  env.init baremetal kubespray air-gapped --skip-issuers --skip-network-policies
+}
+
+setup() {
+  load "../../bats.lib.bash"
+  load_assert
+  load_mock
+
+  mock_kubectl="$(mock_create)"
+  export mock_kubectl
+  kubectl() {
+    # shellcheck disable=SC2317
+    "${mock_kubectl}" "${@}"
+  }
+  export -f kubectl
+}
+
+@test "check_node_label success" {
+  # shellcheck source=bin/common.bash
+  source ../bin/common.bash
```

Comment:
Couldn't source it in `setup()` because then `$config` wasn't defined and `with_kubeconfig` failed. Do any of you know how to fix that?